### PR TITLE
[#217] throwing exception, when imported data contain empty language …

### PR DESCRIPTION
**Pull request by psiotwo**
…tag for string props

### CODE REVIEW

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java [hunk 1]
```diff
@@ -51,6 +50,14 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(SKOSImporter.class);
 
+    private final Set<String> multiLingualProperties = Stream.of(
```
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java]: Why not `static`?

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java [hunk 2]
```diff
@@ -90,6 +97,13 @@ public Vocabulary importVocabulary(final boolean rename,
         }
         LOG.debug("Vocabulary import started.");
         parseDataFromStreams(mediaType, inputStreams);
+        LOG.debug("Checking that only language-tagged literals are provided.");
+
+        final Set<String> languageTags = getLanguageTagsPerProperties(model, multiLingualProperties);
+        if (languageTags.contains("")) {
+            throw new IllegalArgumentException("Each value of the properties must have a non-empty language tag: " + multiLingualProperties);
```
Comment (ledsoft) [inline src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java:104]: Is the `IllegalArgumentException` handled gracefully in the REST exception handling? (Or caught somewhere and replaced with some general import exception)

**Review by ledsoft — DISMISSED** [review]
LGTM, see comments.

File: src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java (hunk 2 above)
Comment (psiotwo) [reply src/main/java/cz/cvut/kbss/termit/persistence/dao/skos/SKOSImporter.java:104]: https://github.com/kbss-cvut/termit/blob/956ba990aab08790730b27f394bfa86e2e846bf6/src/main/java/cz/cvut/kbss/termit/service/repository/VocabularyRepositoryService.java#L188

**Review by ledsoft — APPROVED** [review]

**Pull request merged**